client/src/components/layout/footer.tsx

```diff
@@ -1,5 +1,164 @@
+
+//           <p className={styles.footerNote}>
+//             <FormattedMessage id="footer.builtBy" />
+//             <br />
+//             <FormattedMessage id="footer.copyright" />
+//           </p>
+//         </section>
+
+//         <nav className={styles.footerColumns} aria-label="Footer Navigation">
+//           <section className={styles.footerSection}>
+//             <h3 className={styles.footerHeading}>
+//               <FormattedMessage id="footer.siteMap" />
+//             </h3>
+//             <ul>
+//               <li><a href="/"><FormattedMessage id="footer.homepage" /></a></li>
+//               <li><a href="/maps"><FormattedMessage id="footer.map" /></a></li>
+//               <li><a href="/database"><FormattedMessage id="footer.database" /></a></li>
+//               <li><a href="/resources"><FormattedMessage id="footer.resources" /></a></li>
+//             </ul>
+//           </section>
+
+//           <section className={styles.footerSection}>
+//             <h3 className={styles.footerHeading}>
+//               <FormattedMessage id="footer.importantLinks" />
+//             </h3>
+//             <ul>
+//               <li><a href="#"><FormattedMessage id="footer.bostonBoard" /></a></li>
+//               <li><a href="#"><FormattedMessage id="footer.abcc" /></a></li>
+//               <li><a href="#"><FormattedMessage id="footer.application" /></a></li>
+//               <li><a href="#"><FormattedMessage id="footer.analyzeBoston" /></a></li>
+//             </ul>
+//           </section>
+
+//           <section className={styles.footerSection}>
+//             <h3 className={styles.footerHeading}>
+//               <FormattedMessage id="footer.moreFromUs" />
+//             </h3>
+//             <ul>
+//               <li><a href="#"><FormattedMessage id="footer.offsite" /></a></li>
+//               <li><a href="#"><FormattedMessage id="footer.bootcamp" /></a></li>
+//               <li><a href={links.cod}><FormattedMessage id="footer.codeForBoston" /></a></li>
+//             </ul>
+//           </section>
+//         </nav>
+//       </div>
+//     </footer>
+//   );
+// };
+
+// export default Footer;
+
+
+
+import styles from "./footer.module.css";
+import { FormattedMessage } from "react-intl";
+import { Link } from "@tanstack/react-router"; // or "next/link" if you're using Next.js
+
 const Footer = () => {
-  return <footer>This is the footer content.</footer>;
+  const links = {
+    cod: "https://www.codeforboston.org"
+  };
+
+  return (
+    <footer className={styles.siteFooter} role="contentinfo">
+      <div className={styles.footerContainer}>
+        <section className={styles.footerLeft}>
+          <div className={styles.licenseHeader}>
+            <span className={styles.licenseBadge}>
+              <FormattedMessage id="footer.license" />
+            </span>
+            <span className={styles.licenseTitle}>
+              <FormattedMessage id="footer.toSucceed" />
+            </span>
```

Comment:
Requested Changes:
We want to use the same assets as the header here. That way, any changes will propagate anywhere it is used. Take a look at the [header component and header styles](https://github.com/codeforboston/boston-liquor-license-tracker/tree/main/client/src/components/layout) for more context.
We also do not want to translate our logo.